testsettings.py

```diff
@@ -99,6 +99,20 @@ def _set_logging_levels(levels):
     'loggers': {},
 }
 
+# Required in Python 3 to prevent transient but frequent travis errors.
+# Probably is caused by some race condition.
```

Comment:
This comment sounds pretty hacky. Do you expect to get to the bottom of this at some point?

Reply:
It's as hacky as it sounds.  I don't expect to investigate this much further, but wanted to leave my observations in case someone else comes along with the same issue.